src/conduit/openapi.py

```diff
@@ -22,6 +25,23 @@ def includeme(config: Configurator) -> None:
     config.pyramid_openapi3_add_explorer(route="/api")
 
 
+def object_or_404(obj: t.Optional[t.Any]) -> t.Any:
```

Comment:
maybe `object_or_404(obj: t.Any) -> t.Any` as a parameter is required (atleast `None`).